pkg/cmd/extension/manager.go

```diff
@@ -469,9 +469,18 @@ func (m *Manager) Upgrade(name string, force bool) error {
 }
 
 func (m *Manager) upgradeExtensions(exts []*Extension, force bool) error {
+	var longestExtName = 0
+	for _, ext := range exts {
+		l := len(ext.Name())
+		if len(ext.Name()) > longestExtName {
+			longestExtName = l
+		}
+	}
```

Comment:
Could also seeing this use [`slices.Max`](https://pkg.go.dev/slices#Max) or [`slices.MaxFunc`](https://pkg.go.dev/slices#MaxFunc) 🤔

Reply:
done in https://github.com/cli/cli/pull/10216